portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,15 +19,37 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String greeting = "Hello Autumn!";
-    response.setContentType("text/html;");
-    response.getWriter().println(greeting);
+      ArrayList<String> comments = new ArrayList<String>();
+    comments.add("sup Autumn");
+    comments.add("What other projects have you done");
+    comments.add("hi");
+    String json = convertToJSON(comments);
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+  }
+
+  private String convertToJSON(ArrayList<String> comments){
+    String json = "{";
+    json += "\"comment1\": ";
+    json += "\"" + comments.get(0) + "\"";
+    json += ", ";
+    json += "\"comment2\": ";
+    json += "\"" + comments.get(1) + "\"";
+    json += ", ";
+    json += "\"comment3\": ";
+    json += comments.get(2);
```

Comment:
Does this work? There are no quotes around this string.

Reply:
fixed!